### PR TITLE
Module1 Branch created for final tauch of Awesome Book Project

### PR DESCRIPTION
**Pull request by SAZZAD1Q2**
Added Navigation bar.
    Made three content sections.
      1.  Books list.
      2. Add book form.
      3. Contact info.

### CODE REVIEW

File: modules/updateTime.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  const now = DateTime.now();
+  const options = { month: 'short', day: 'numeric', year: 'numeric' };
+  const date = now.toLocaleString(options);
+  const time = now.toLocaleString(DateTime.TIME_WITH_SECONDS);
+  document.getElementById('dateTime').innerHTML = `${date} , ${time}`;
+};
+
+setInterval(updateTime, 1000);
+
+export default updateTime;
```
Comment (GzimAsani) [inline modules/updateTime.js:13]: - [ ] Kindly add this on the right side of the app just like the given wireframe:

Yours:
image

Design:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @SAZZAD1Q2 and @AbdusSattar-70               :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with styling, it looks great :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/updateTime.js (hunk 1 above)
Comment (SAZZAD1Q2) [reply modules/updateTime.js:13]: Hello
Thank you so much for the suggestion. I actually forgot to do so. Now I am setting the dateTime right side to submit again.
Thank you once again.
Sazzad

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @SAZZAD1Q2, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**